docs/source/sections/webapp/releases.rst

```diff
@@ -2,12 +2,45 @@
 Web-App Release Notes
 *********************
 
+* **February 1, 2018** 
+
+ **Please Note:** Work is underway to rework our cohort creation page to better differentiate between samples which are from image data vs. those which are not.
+ 
+ **Issues resolved in Sprint 23 as of 02/01/2018**
+ 
+ New Enhancements
+  - We have enabled DNA methylation data to be used when plotting with genomic build hg38.
+  - The cohort view files page has been updated to File Browser. The File Browser page also now has new filters data level, data type, and disease code. A time stamp has also been added to the CSV file that can be downloaded.
+  - The IGV browser and caMicroscope are now more clearly defined and separated on the File Browser page. 1931
+  - When uploading a set of barcodes to create a cohort the error message has been redefined to be more general.
+  
+ Bug Fixes
+  - You can now plot DNA methylation data using genomic build hg19 when working on a worksheet.
+  - When registering a service account to controlled data you will no longer receive an error message when certain Google managed service accounts are also on the IAM and Admin page.
+  - On a  worksheet, if you add new cohorts to a worksheet with pre-existing cohorts. Now the older and newly added cohorts are present on the worksheet for analysis.
+  - When working with a worksheet you are now able to plot gene names that contain periods.
+ 
+ **Known Issues in Sprint 23 as of 02/01/2018**
+  - Analysis Type: Seq peek Formatting Elongated on occasion.
+  - If the user shares a Cohort neither the owner nor the person who was granted access to Cohort will receive a confirmation email when sharing a Cohort. 
+  - CCLE data cannot be plotted when working with workbooks.  ISB-CGC will resolve this functionality after the GDC formally releases CCLE data.
+  - When a user duplicates a Worksheet, then tries to implement the log scale it will not function properly.
+  - The set operation for existing Cohorts complement is behaving exceptionally slow.
+  - The mouse-over feature is currently disabled for program TARGET with disease code ALL.
+  - When working on Firefox browser a violin plot does not display the data plotted correctly when working on a Worksheet.
+  - When uploading TARGET files using the cohort barcode creation feature from the GDC you may get an invalid barcodes error message and unable to upload all the barcodes.
+  - When using SeqPeek with data other then TCGA, the BQ table used list will display that only TCGA tables where searched.
```

Comment:
This might be misleading--SeqPeek can't actually be used with other programs. Probably better to reword 'SeqPeek can only be plotted with TCGA data, but if a cohort contains no TCGA samples the SeqPeek analysis will still search the TCGA BigQuery tables.'